public/documents/fari-rpgs/charge-rpg.md

```diff
@@ -883,13 +883,11 @@ We aren't using dice to support this mechanic because we don't believe luck shou
 
 **_The future is in constant flux._**
 
-Not everything is about the proficiencies of the PCs. There are situations where something is simply up to chance. In those situations, the GM can call for a `fortune roll`.
+The fortune roll is a tool that the GM can use to disclaim decision making in a situation the where PCs aren’t directly involved in. It can also be used by the players when no other roll applies to the situation at hand. The result of the roll helps guide the outcome as to where the story goes next, but doesn't feature `momentum` nor `consequences`.
```

Comment:
Extra word

```suggestion
The fortune roll is a tool that the GM can use to disclaim decision making in a situation the PCs aren’t directly involved in. It can also be used by the players when no other roll applies to the situation at hand. The result of the roll helps guide the outcome as to where the story goes next, but doesn't feature `momentum` nor `consequences`.
```